apps/openmw/mwbase/world.hpp

```diff
@@ -566,6 +573,8 @@ namespace MWBase
             virtual void setPlayerTraveling(bool traveling) = 0;
             virtual bool isPlayerTraveling() const = 0;
 
+            virtual void travel(TravelTarget destination) = 0;
```

Comment:
inb4 cppcheck complains that TravelTarget can be passed by (const) reference.
Anyway it might be better to use ```travel(const CellID&, const ESM::Position&)```. That way we don't need to declare that TravelTarget struct, and by using CellID we won't have to change this code when extra worldspaces become supported in the future.